src/components/behandlerdialog/meldinger/Samtaler.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useMemo } from "react";
+import { useBehandlerdialogQuery } from "@/data/behandlerdialog/behandlerdialogQueryHooks";
+import {
+  Conversations,
+  Melding,
+} from "@/data/behandlerdialog/behandlerdialogTypes";
+import { Accordion } from "@navikt/ds-react";
+import { tilDatoMedManedNavnOgKlokkeslett } from "@/utils/datoUtils";
+import styled from "styled-components";
+
+const StyledSamtaler = styled.div`
+  margin-top: 1.5em;
+`;
+
+const sortMeldingerByTidspunkt = (m1: Melding, m2: Melding) => {
+  return new Date(m2.tidspunkt).getTime() - new Date(m1.tidspunkt).getTime();
+};
+
+const sortConversations = (conversations: Conversations): Melding[][] => {
+  const conversationRefs: string[] = Object.keys(conversations);
+  conversationRefs.sort((a, b) => {
+    const aNewestMelding: Melding = conversations[a].sort((m1, m2) =>
+      sortMeldingerByTidspunkt(m1, m2)
+    )[0];
+    const bNewestMelding: Melding = conversations[b].sort((m1, m2) =>
+      sortMeldingerByTidspunkt(m1, m2)
+    )[0];
+    return sortMeldingerByTidspunkt(aNewestMelding, bNewestMelding);
+  });
+
+  const sortedMeldinger: Melding[][] = [];
+  conversationRefs.forEach((ref: string) => {
+    sortedMeldinger.push(conversations[ref]);
+  });
+  return sortedMeldinger;
+};
+
+export const Samtaler = () => {
+  const { data } = useBehandlerdialogQuery();
+
+  const sortedConversations = useMemo(() => {
+    return data ? sortConversations(data.conversations) : [];
+  }, [data]);
```

Comment:
Prøvde meg på en `useMemo` her for å slippe å gjøre sorteringen oftere enn man trenger. Troor det skal være gunstig her 🤔